composer.json

```diff
@@ -9,7 +9,7 @@
         "drupal/webform": "^5.2"
     },
     "suggest": {
-        "dpc-sdp/tide_api:^1.2.6": "Allows to use Drupal in headless mode"
+        "dpc-sdp/tide_api": "^1.2.9"
```

Comment:
@vincent-gao Can you please add the description back?
We want to retain the previous format of `package:version:description`. During install suggests prints out the description to provide some context.

Reply:
thanks @GROwen !